src/plugins/bubble/bubble.js

```diff
@@ -0,0 +1,165 @@
+
+	/**
+	 * Initiate chat wizard bubble
+	 * @method adjustBubbleMargin
+	 * @param {jQuery DOM element} $selector Element which is the actual bubble
+	 */
+	adjustBubbleMargin = function( $selector ) {
+
+		var $footer = $( "#wb-info" );
+
+		// Hide basic form on load, show chat bubble instead
+		$selector.fadeIn( "slow" );
+
+		// Ensure that the bubble does not go passed the footer
+		if ( $footer.length ) {
+
+			// Add some white space over the footer for the bubble to sit
+			$footer.addClass( componentName + "-mrgn" );
+
+			// Keep the bubble sticky while scrolling Y until user reaches the footer
+			var stickyUntilFooter = function( $element ) {
+
+			// Equals to bubble default bottom value in CSS
+				var bottomY = $( "main p" ).eq( 0 ).height() || 30;
+				var $window = $( window );
+
+				if ( $window .scrollTop() >= $document.outerHeight() - $window .outerHeight() - $footer.outerHeight() ) {
+					$element.css( {
+						bottom: ( $footer.outerHeight() - ( $document.outerHeight() - $window.outerHeight() - $window.scrollTop() ) + bottomY )
+					} );
+				} else {
+					$element.css( {
+						bottom: bottomY
+					} );
+				}
+			};
+
+			// Correct bubble positionning on load, on resize an on Y scroll if necessary
+			stickyUntilFooter( $selector );
+
+			$( window ).on( "resize scroll", function() {
+				stickyUntilFooter( $selector );
+			} );
+		} else {
+			$( "main" ).after( "<footer id=\"wb-info\" class=\"" + componentName + "-mrgn" + "\"></footer>" );
+		}
+	},
+
+
+	setNotificationStatusWithExpiry = function( key, ttl ) {
+		var now = new Date();
+		localStorage.setItem( key, now.getTime() + ttl );
+	},
+
+	getNotificationStatusWithExpiry = function( key ) {
+		var expiryTimeValue = localStorage.getItem( key );
+		var now = new Date();
+
+		// compare the expiry time of the item with the current time
+		if ( now.getTime() > expiryTimeValue ) {
+
+			// If the item is expired, delete the item from storage
+			// and return null
+			localStorage.removeItem( key );
+			return null;
+		}
+		return expiryTimeValue;
+	},
+
+
+/**
+	 * @method init
+	 * @param {jQuery Event} event Event that triggered the function call
+	 */
+	init = function( event ) {
+
+		// Start initialization
+		// returns DOM object = proceed with init
+		// returns undefined = do not proceed with init (e.g., already initialized)
+		var elm = wb.init( event, componentName, selector ),
+			$elm;
+		if ( elm ) {
+			$elm = $( elm );
+
+			var $bubbleElm = $( "<div class='" + componentName + "-wrap'></div>" );
+			$elm.wrap( $bubbleElm );
+
+			// Get notification status to check if it is still valid after TTL
+			isNotif = getNotificationStatusWithExpiry( componentName + "-notif" );
+
+			var $notification = ( !isNotif ? "<p class=\"trans-left\">\r\n" +
+			"<span class=\"notif\">" + $elm.text() + "</span>\r\n" +
+			"<button class=\"notif-close\" title=\"Close notification\" aria-label=\"Close notification\" >×</button>" : "" );
```

Comment:
For this and the skip link and here, please add a i18n mechanism to display the right text in the right language.

In the skip links, "Skip to" becomes "Passer à" in French. And here, "Close notification" becomes "Fermer la notification".

Look in the chat wizard code for that i18n mechanism.

Reply:
Also, the close button for the notification doesn't seem to work as expected.

Reply:
the click action doesn't work on button, but works on anchor <a>